docs/recipes/WritingTests.md

```diff
@@ -258,46 +258,38 @@ describe('todos reducer', () => {
 
 A nice thing about React components is that they are usually small and only rely on their props. That makes them easy to test.
 
-First, we will install [Enzyme](http://airbnb.io/enzyme/). Enzyme uses the [React Test Utilities](https://facebook.github.io/react/docs/test-utils.html) underneath, but is more convenient, readable, and powerful.
+First, we will install [React Testing Library](https://testing-library.com/docs/react-testing-library/intro). React Testing Library is built on top of React-DOM and React-DOM/test-utils, but is more convenient, readable, and powerful.
 
 ```sh
-npm install --save-dev enzyme
+npm install --save-dev @testing-library/react
```

Comment:
I'd also recommend having people install `@testing-library/jest-dom` if you're suggesting people use `jest`.